fuse/utils/misc/misc.py

```diff
@@ -118,9 +118,9 @@ def batch_dict_to_string(batch_dict: dict) -> str:
         :return: string representation of the batch dict
         """
         res = ''
-        all_keys = FuseUtilsHierarchicalDict.get_all_keys(batch_dict)
```

Comment:
@mosheraboh 
In the description it says that: `:param batch_dict: might be any dict`,  but there is a use of `FuseUtilsHierarchicalDict` class method.

I think such a design is not optimal so I suggest to make sure that the dictionary is a NDict instance and use instance methods or open a new utils class for such cases.


Any thoughts? 🙂